examples/optional-round-trip-time-with-sonar-availability.ts

```diff
@@ -0,0 +1,93 @@
+// Optimal Round Trip Time with Sonar Availability
+// Main configuration
+const configuration = {
+    /** List of  providers configuration*/
+    providers: [
+        {
+            name: ('jsdelivr-cdn' as TCDNProvider), // CDN Provider alias to work with
+            monitorId: (301 as TMonitor), // Monitor ID which is created by user to monitor hostname
+            cname: 'www.foo.com' // cname to pick as a result
+        },
+        {
+            name: ('cloudflare' as TCDNProvider),
+            monitorId: (302 as TMonitor),
+            cname: 'www.bar.com'
+        },
+        {
+            name: ('google-cloud-cdn' as TCDNProvider),
+            monitorId: (303 as TMonitor),
+            cname: 'www.baz.com'
+        }
+    ],
+    defaultTtl: 20, // The DNS TTL to be applied to DNS responses in seconds.
+    availabilityThreshold: 90 // Board value for providers to compare with
+};
+/**
+ * returns index of Highest value in array
+ * @param array
+ */
+const getHighest = (array: number[]): number => array.indexOf(Math.max(...array));
+/**
+ * returns object which have highest value of property
+ * @param array
```

Comment:
@HesyRa types not need to define in JS docs, we already have them defined in syntax

Reply:
@HesyRa please review all PRs and do this changes.